### PR TITLE
Sprint1 jono

### PR DESCRIPTION
**Pull request by Jaunault**

### CODE REVIEW

File: backend/store/admin.py [hunk 1]
```diff
@@ -313,17 +324,17 @@ def get_list_filter(self, request):
 admin.site.register(MenuOptionProduct, MenuOptionProductAdmin)
 
 
-class TaxRateAdmin(FilterUserAdmin):
+class TaxRateAdmin(ModelAdmin):
     def changelist_view(self, request, extra_context=None):
         if request.user.is_superuser:
-            self.list_display = ('name', 'tax_rate', 'store')
+            self.list_display = ('name', 'tax_rate')
         else:
```
Comment (jproject-fr) [inline backend/store/admin.py:331]: No need to make a difference if both are the same

File: backend/store/admin.py [hunk 2]
```diff
@@ -313,17 +324,17 @@ def get_list_filter(self, request):
...
             self.list_display = ('name', 'tax_rate')
         return super(TaxRateAdmin, self).changelist_view(request, extra_context)
 
     def get_fields(self, request, obj=None):
         if request.user.is_superuser:
-            return ['name', 'tax_rate', 'store']
+            return ['name', 'tax_rate']
```
Comment (jproject-fr) [inline backend/store/admin.py:337]: Same here, no need to make a difference if it's the same

File: backend/store/migrations/0005_voucher.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                ('single_use_only', models.BooleanField(help_text='If you check this box, the promotional code can be used only one time per user account', verbose_name='single use only')),
+                ('display_on_the_shopping_cart', models.BooleanField(default=False, help_text='If you check this box, the promotional code and the description will be displayed on the shopping cart of the user, otherwise it will be hid', verbose_name='display on the shopping cart')),
+                ('valid_from', models.DateField(verbose_name='valid from')),
+                ('valid_to', models.DateField(verbose_name='valid to')),
+                ('is_active', models.BooleanField(default=False, help_text='If you check this box, the code will be activated', verbose_name='is active')),
+                ('belongs_to_store', models.ForeignKey(default=1, on_delete=django.db.models.deletion.CASCADE, to='store.store', verbose_name='store')),
```
Comment (jproject-fr) [inline backend/store/migrations/0005_voucher.py:29]: Confusing : set store instread of belongs_to_store

File: backend/store/models.py [hunk 4]
```diff
@@ -381,4 +376,63 @@ class Meta:
...
+    )
+    valid_to=models.DateField(
+        verbose_name=_('valid to')
+    )
+    is_active=models.BooleanField(
+        default=False, 
+        verbose_name=_('is active'), 
+        help_text=_("If you check this box, the code will be activated")
+    )
+    belongs_to_store = models.ForeignKey(
```
Comment (jproject-fr) [inline backend/store/models.py:427]: replace with store

**Comment by jproject-fr** [comment]
Add permissions for Restaurants to have access to the tax rate

**Comment by Jaunault** [comment]
@jproject-fr Correction made

**Pull request merged**